src/controller/controller.ts

```diff
@@ -734,12 +734,8 @@ class Controller extends events.EventEmitter {
             } else {
                 /* istanbul ignore else */
                 if (frame.header.isSpecific) {
-                    if (Events.CommandsLookup[command.name]) {
-                        type = Events.CommandsLookup[command.name];
-                        data = frame.payload;
-                    } else {
-                        logger.debug(`Skipping command '${command.name}' because it is missing from the lookup`, NS);
-                    }
+                    type = `command${command.name.charAt(0).toUpperCase()}${command.name.slice(1)}`;
```

Comment:
What was the initial drive behind using `command${}` instead of just using `command.name`?
`MessagePayloadType` also probably needs an overhaul.

Reply:
Because these are commands, in this way they never can conflict with e.g. `read` and `message`. For the `MessagePayloadType` we maybe should put `command` there and put the command name on another property, but that's for later.

Reply:
I guess we can do this until then, to atleast narrow the typing a bit:
```typescript
type MessagePayloadType = 'attributeReport' | 'readResponse' | 'raw' | 'read' | 'write' | `command${string}`;
```

Reply:
Didnt know this was possible, cool!